### ISSUE TITLE
[$30] [Edge] The NVDA still read item name when user try to mouse hover outside area of them

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3072]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Try to mouse hover outside area of the "footer-menu"

**Expected Result**
The NVDA should not read item name when user try to mouse hover outside area of them

**Actual Result**
The NVDA still read item name when user try to mouse hover outside area of them

**Device**: Desktop/Labtop
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6b62afda%2Dc894%2D4ce2%2Da924%2D74186e1c15dc&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fdoremihong%2FBug%5F11

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors open for pickup. PRs against `qa-accessibility`.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097750 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097750 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097750 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@codeMinter @veshu unable to reproduce this. 

Test URL - https://www.topcoder.com/challenges
Env - Windows/Edge
Screen Reader Used - NVDA

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
